test/.eslintrc.yml

```diff
@@ -0,0 +1,43 @@
+env:
+  es6: true
+  node: true
+
+parserOptions:
+  ecmaVersion: 6
+  sourceType: module
+  ecmaFeatures:
+    impliedStrict: true
+
+parser: babel-eslint
+
+plugins:
+  - ava
+
+extends: "eslint:recommended"
+
+rules:
+  ava/max-asserts: [0, 5]
```

Comment:
Cannot extend AVA recommended — plugin is broken and not yet published. May be in the future.